amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-reference.adoc

```diff
@@ -0,0 +1,3652 @@
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  `#[payload]` |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+
+[%autowidth.spread]
+|===
+|Type |<<BucketVersioningResponse>>
+|===
+
+==== Associated Configurations
+
+* <<config>>
+
+==== Throws
+
+* S3:BAD_REQUEST
+* S3:CONNECTIVITY
+* S3:FORBIDDEN
+* S3:RETRY_EXHAUSTED
+* S3:S3_ERROR
+* S3:SERVER_BUSY
+* S3:SERVICE_ERROR
+
+
+[[GetBucketWebsite]]
+=== Get Bucket Website
+`<s3:get-bucket-website>`
+
+
+Returns the website configuration for a bucket. To host a website on Amazon S3, you can configure a bucket as a website by adding a website configuration. This GET action requires the S3:GetBucketWebsite permission. By default, only the bucket owner can read the bucket website configuration. However, bucket owners can allow other users to read the website configuration by writing a bucket policy granting them the S3:GetBucketWebsite permission.
+
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Bucket a| String |  Name of the bucket. |  | x
+| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  `#[payload]` |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+
+[%autowidth.spread]
+|===
+|Type |<<BucketWebsiteResponse>>
+|===
+
+==== Associated Configurations
+
+* <<config>>
+
+==== Throws
+
+* S3:BAD_REQUEST
+* S3:CONNECTIVITY
+* S3:FORBIDDEN
+* S3:RETRY_EXHAUSTED
+* S3:S3_ERROR
+* S3:SERVER_BUSY
+* S3:SERVICE_ERROR
+
+
+[[GetObject]]
+=== Get Object
+`<s3:get-object>`
+
+
+Retrieves objects from Amazon S3. To use GET, you must have READ access to the object. If you grant READ access to the anonymous user, you can return the object without using an authorization header. If the object you are retrieving is stored in the S3 Glacier or S3 Glacier Deep Archive storage class, or S3 Intelligent-Tiering Archive or S3 Intelligent-Tiering Deep Archive tiers, before you can retrieve the object you must first restore a copy using the *Restore Object* operation. Otherwise, this action returns an InvalidObjectState error. For more information about restoring archived objects, refer to https://docs.aws.amazon.com/AmazonS3/latest/dev/restoring-objects.html[Restoring Archived Objects].
```

Comment:
```suggestion
Retrieves objects from Amazon S3. To use GET, you must have READ access to the object. If you grant READ access to the anonymous user, you can return the object without using an authorization header. If the object you are retrieving is stored in the S3 Glacier or S3 Glacier Deep Archive storage class, or S3 Intelligent-Tiering Archive or S3 Intelligent-Tiering Deep Archive tiers, before you can retrieve the object. you must first restore a copy using the *Restore Object* operation. Otherwise, this action returns an InvalidObjectState error. For more information about restoring archived objects, refer to https://docs.aws.amazon.com/AmazonS3/latest/dev/restoring-objects.html[Restoring Archived Objects].
```